lldb/source/Plugins/ObjectFile/Minidump/MinidumpFileBuilder.cpp

```diff
@@ -655,9 +655,10 @@ MinidumpFileBuilder::AddMemoryList(const lldb::ProcessSP &process_sp,
     const addr_t addr = core_range.range.start();
     const addr_t size = core_range.range.size();
     auto data_up = std::make_unique<DataBufferHeap>(size, 0);
+    Status read_error;
```

Comment:
The nit-pickiest thing I can come up with is "why you need a new Status variable?". Good job on following the LLDB naming conventions for locals (which are *different* from the LLVM naming conventions for locals, so you have to be aware of what part of the code base you're working in..)

Reply:
This is a good question. I originally thought about returning Status{}. However, conceptually it is wrong imo to use the same error status for ReadMemory() as the error status used for handling the functions errors.